src/dataCollection.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+"""
+@file dataCollection.py .
+
+@author Ayush Saini
+@email ayush@robrosystems.com
+@brief script for Data Collection using ROS Topic
+@version 0.1.0
+@date 2021-08-28
+@copyright Copyright (c) 2021
+"""
+import os
+
+import cv2
+
+from cv_bridge import CvBridge, CvBridgeError
+
+from google.cloud import storage
+
+from robro_cloud.robroBucketImageUploader import RobroBucketImageUploader
+
+import rospy
+
+from sensor_msgs.msg import Image
+
+from std_msgs.msg import String
+
+rospy.init_node('dataCollection', anonymous=True)
+
+proj_name = rospy.get_param('~proj_name')
+ros_topic_name = rospy.get_param('~ros_topic_name')
```

Comment:
Please give proper naming for `ros_topic_name`.

Reply:
done